Assets/Scripts/GrappleGunBehavior.cs

```diff
@@ -101,11 +124,13 @@ private void FixedUpdate()
     {
         ReticleEffect();
         if (!grappling) return;
-        if (GrappleBroken()) StopGrapple(); // TODO: add particle effects, SFX, and/or animation to indicate what happened
+        if (breakable && GrappleBroken()) StopGrapple(); // TODO: add particle effects, SFX, and/or animation to indicate what happened
         else
         {
+            joint.maxDistance *= 0.975f;
```

Comment:
Shouldn't `maxDistance` just be set to the distance between the player and asteroid, so that it can never grow?